src/card.css

```
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 400;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
```

center card with flex (add right property to flex container)

```suggestion
  margin-bottom: 40px;
```